src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -48,7 +48,7 @@ public static class PermutationFeatureImportanceExtensions
         /// <param name="catalog">The regression catalog.</param>
         /// <param name="predictionTransformer">The model on which to evaluate feature importance.</param>
         /// <param name="data">The evaluation data set.</param>
-        /// <param name="labelColumnName">Label column name.</param>
+        /// <param name="labelColumnName">Label column name.  The column data must be <see cref="System.Single"/>.</param>
```

Comment:
>data must be <see cref="System.Single"/>. [](start = 73, length = 41)

nit: The column's data type must be a scalar of <see cref="System.Single"/>
Applies to other places as well.

Reply:
Did we say this for the other scalar types?

Reply:
No.

---
In reply to: [277187743](https://github.com/dotnet/machinelearning/pull/3455#discussion_r277187743) [](ancestors = 277187743)

Reply:
Yes, I think keep it consistent